### DISCUSSION TITLE
Add a calendar

### DISCUSSION THREAD

**Discussion by snomie** [SiimKirjanen/seatreg#61] [Q&A] [answered]
Good morning.
I'm trying this plugin, it's great!
I just have one question, how can I create a calendar to manage multi-day bookings? Thank you!

**Comment by SiimKirjanen** [comment, marked as answer]
Hey.
Currently there is no calendar functionality.

**Comment by snomie** [comment]
OK thank you.  However I can pay you to implement the calendar.  If okay
send me a private message with the cost

Il dom 18 set 2022, 17:02 Siim Kirjanen ***@***.***> ha
scritto:

**Comment by SiimKirjanen** [comment]
Added with version 1.33.0
